.travis.yml

```diff
@@ -0,0 +1,22 @@
+sudo: false
+language: node_js
+cache:
+  directories:
+    - node_modules
+notifications:
+  email: false
+node_js:
+  - '6'
+  - '4'
+before_install:
+  - npm i -g npm@^2.0.0
```

Comment:
That came with the semantic release `setup` command. what do you think, what should we do here now?